activities/beginner/using-fetch/README.md

```diff
@@ -0,0 +1,5 @@
+# Using Fetch
+
+Fetch is a modern browser implementation for fetching data -- think a new alternative to Ajax. Try fetching some data from an API and using that to alter your HTML.
```

Comment:
I think this is a great description - I would add a brief sentence along the lines of "An introduction to Fetch, a modern browser implementation...". See comment on the Closure activity - this is the type of description containing basic context that I feel would benefit people viewing these for the first time.